storage/object/api-cli/installing-rclone.mdx

```diff
@@ -0,0 +1,341 @@
+---
+title: Installing and setting up Rclone
+description: This page explains how to installing and set up Rclone.
+excerpt: |
+  This page explains how to installing and set up Rclone.
+image: /assets/images/docs/scaleway.png
+keywords: cloud tutorials, cloud documentations, cloud instance, cloud hosting, server, storage, s3, object-storage, aws-cli, multipart upload, aborting multipart upload, api, cli, minio, minio client, cleaning up multipart upload
+dates:
+  validation: 2021-08-30
+  posted: 2021-08-25
+---
+
+[Rclone](https://rclone.org) is a command-line tool that can be used to manage your cloud storage. It communicates with any S3 compatible cloud storage provider as well as other storage platforms.
+
+Follow the instructions given in the [official rclone documentation here](https://rclone.org/install/) to install rclone. 
+
+    For example, on Linux:
+
+1. Connect to your server as `root` via [SSH](/compute/instances/how-to/connect-to-instance/).
+
+2. Update the APT packet cache and the software already installed on the instance:
+  ```
+  apt update && apt upgrade -y
+  ```
+
```

Comment:
I think we can remove this step. It is not relevant to the tutorial, as `rclone` is installed as a standalone binary from a direct URL.